src/createServer.js

```diff
@@ -1,3 +1,25 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { isValidReq } = require('./isValidRequest');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const text = req.url.split('?')[0].replace('/', '');
+    const search = req.url.split('?')[1].split('=');
+    const [searchName, searchParam] = search;
+
+    const { error, message } = isValidReq(text, searchName, searchParam);
+
+    const contentRes = !error
+      ? convertToCase(text, searchParam)
+      : { errors: [{ message }] };
+
+    res.end(JSON.stringify(contentRes));
```

Comment:
Set the 'Content-Type' header to 'application/json' to indicate that the response is a JSON object.